packages/clickable/components/clickable.tsx

```diff
@@ -6,7 +6,7 @@ export interface ClickableProps {
   /**
    * Children should be a HTML element.
    */
-  children: React.ReactElement<HTMLElement>;
+  children: React.ReactElement<HTMLElement> & React.ReactNode;
```

Comment:
Was there a type error that came up requiring you to add this `& React.ReactNode`?

Reply:
Yes